client/blocks/taxonomy-manager/index.jsx

```diff
@@ -11,27 +12,70 @@ import SectionHeader from 'components/section-header';
 import Card from 'components/card';
 import SearchCard from 'components/search-card';
 import Button from 'components/button';
+import TermsList from './list';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getPostTypeTaxonomy } from 'state/post-types/taxonomies/selectors';
+import QueryTaxonomies from 'components/data/query-taxonomies';
 
 export class TaxonomyManager extends Component {
 	static propTypes = {
-		translate: PropTypes.func,
 		taxonomy: PropTypes.string,
+		labels: PropTypes.object,
+		postType: PropTypes.string,
+		siteId: PropTypes.number,
 	};
 
+	static defaultProps = {
+		postType: 'post'
+	};
+
+	constructor( props ) {
+		super( props );
+		this.state = {
+			search: null
+		};
+
+		this.onSearch = this.onSearch.bind( this );
```

Comment:
This is redundant with the property initializer below, you'd only need one or the other to keep the function bound to the current instance.